roles/ceph-nfs/defaults/main.yml

```diff
@@ -46,23 +52,24 @@ ceph_nfs_rgw_user: "cephnfs"
 #ceph_nfs_rgw_access_key: "QFAMEDSJP5DEKJO0DDXY"
 #ceph_nfs_rgw_secret_key: "iaSFLDVvDdQt6lkNzHyW4fPLZugBAI1g17LO0+87[MAC[M#C"
 
-
 ###################
 # CONFIG OVERRIDE #
 ###################
 
 # Ganesha configuration file override.
-# This allows you to specify more configuration options
-# using an INI style format.
-# The following sections are supported: [global], [mon], [osd], [mds], [rgw]
+# This multiline string will be appended to the contents of ganesha.conf and
+# must be in the correct ganesha.conf format seen here:
+# https://github.com/nfs-ganesha/nfs-ganesha/blob/next/src/config_samples/ganesha.conf.example
 #
 # Example:
-# ceph_conf_overrides:
-#   global:
-#     foo: 1234
-#     bar: 5678
+#CACHEINODE {
+        #Entries_HWMark = 100000;
+#}
 #
-ganesha_conf_overrides: {}
+#ganesha_conf_overrides: |
+#    CACHEINODE {
```

Comment:
You need a call a section if you want to override the template.

Reply:
I'm not trying to override the template, I'm trying to add additional sections to at the person who's configuring the node's discretion. Correct me if I'm wrong, but I can't override the template the way it's described here (https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-nfs/tasks/pre_requisite.yml#L36) since Ganesha's config isn't in INI format or any of the other format's the config_template action can take in.

Reply:
Oh right, this sounds reasonable to me then.